controllers/clusteroperator.go

```diff
@@ -121,6 +137,24 @@ func getOperandVersions() []osconfigv1.OperandVersion {
 	return operandVersions
 }
 
+// printOperandVersion prints current version as a string
+func (r *ProvisioningReconciler) printOperandVersions() string {
+	versionsOutput := []string{}
+	for _, operand := range getOperandVersions() {
+		versionsOutput = append(versionsOutput, fmt.Sprintf("%s: %s", operand.Name, operand.Version))
+	}
+	return strings.Join(versionsOutput, ", ")
+}
+
+// getCurrentVersions returns current version
+func (r *ProvisioningReconciler) getCurrentVersions() []osconfigv1.OperandVersion {
+	co, err := r.getOrCreateClusterOperator()
+	if err != nil {
```

Comment:
I guess returning the defaults is expected, but should we at least log here?

Reply:
+1. Fixed it. Thanks.

Reply:
I don't see getCurrentVersions() being called from anywhere.

Reply:
Yep. Not needed in the PR. Going to remove it.